src/server/services/list/controller.js

```diff
@@ -37,6 +37,7 @@ const servicesListController = {
         headers: [
           { id: 'owner', classes: 'app-entity-table__cell--owned' },
           { id: 'service', text: 'Service', width: '20', isLeftAligned: true },
+          { id: 'tags', text: 'Tags', width: '5' },
```

Comment:
FYI to be responsive - the widths should add up to a total of 100. I suspect this may look a little off on some screens?